Rakefile

```diff
@@ -4,6 +4,7 @@
 require File.expand_path('../config/application', __FILE__)
 require 'rake'
 require 'ci/reporter/rake/minitest' if Rails.env.test?
+require 'ci/reporter/rake/rspec'
```

Comment:
No worries, only reason I know is because I was burned by trying to deploy a couple of apps that had already had this change made.